coalib/bearlib/aspects/__init__.py

```diff
@@ -54,6 +54,21 @@ def __init__(self, module):
                 subaspect = getattr(submod, submodname)
                 setattr(self, submodname, subaspect)
 
+    def get(self, aspectname, multi_find=False):
+        """
+        Wrapper for ``__getitem__`` that doesn't raise exception for searching
```

Comment:
Don't name it a wrapper for `__getitem__`. Not interesting for the user. Just explain what it does, or at least change it to something like `Similar to __getitem__, but ...`